Feature branch

zdrale opened the pull request:
This is the second project in the Microverse main curriculum. It is a clone of the original Mint's Sign Up page. We used Html and CSS for page layout. This branch includes a project index file and styles.

zdrale committed:
First Commit

zdrale committed:
Added markup for page from

zdrale committed:
Style buttons

sinansevgi committed:
Style added for buttons

Images for design added to the 'img' folder

zdrale committed:
Added logos

Styled footer

zdrale committed:
Update README.md

zdrale committed:
Added screenshot file

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,77 @@
+      <a href="">I forgot my userID or Password</a>
+    </div>
+    <div>
+      <button class="sign-in-button"><img class="signin-logo" src="img/signinlogo.png" alt=""> Sign In</button>
```
- [x] the button is not click-able please make the changes so that it looks exactly like on the [original page](https://accounts.intuit.com/signup.html).

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,77 @@
+      <div class="main-logo">
+          <img src="img/mainlogo.png" alt="main-logo">
+      </div>
+      <div class="side-logos">
+       <img src="img/mint.png" alt="nmint-logo">
+       <img src="img/quick.png" alt="quick-logo">
+       <img src="img/turbo.png" alt="turbo-logo">
+      </div>
```
Please follow the following guidelines
- [x] It shows the Mint logo and text (logo on the left, text on the right)

- [x] It shows the Quickbooks logo and text (logo on the left, text on the right)

- [x]  It shows the Turbotax logo and text (logo on the left, text on the right)

zdrale replied:
On the original page, they used SVG files, not text, so we could not find the original font so we used the original SVG file that we found on mint's login page.

arslanbisharat reviewed (changes requested):
### Request changes
I can see you worked hard on this project but I am sorry to inform you that I cannot Approve this project because it is not completed yet.😟
_It's okay. We are here to learn. Don't get discouraged. Work more hard._
**Please have a look at the suggestions to get more understanding. 👀**

- [x] Your PR is not descriptive Please tell us what you did in this project dhow you did. Is there anything else you want to tell TSEs about this project?

- [x] Please make your readme more descriptive. Try to add more social links like Linkedin and Twitter. Also, add something about the project and repo e.g, how to use it. 

- [x] Try to increase the font-size of the text. Please have a look at the original website.
**Orignal**
![chrome-capture (24)](https://user-images.githubusercontent.com/58504655/93438183-63dfcf80-f8e6-11ea-96fc-166f82c76f7f.jpg)
**Your work** 
![chrome-capture (23)](https://user-images.githubusercontent.com/58504655/93438189-6510fc80-f8e6-11ea-96bf-b9f9a173573f.jpg)

- [x]  This button is not clickable. I mean pointer should change as it changes on the original page. 
![chrome-capture (26)](https://user-images.githubusercontent.com/58504655/93439082-8a523a80-f8e7-11ea-8f1b-e103494a2f2c.jpg)


- [x] You used images in the place of text. Please change these image text to text. 
![chrome-capture (25)](https://user-images.githubusercontent.com/58504655/93438706-1adc4b00-f8e7-11ea-8916-11aab2960c6a.jpg)

- [ ] 

- [x] **[Optional]** Please add a valid description in about section of your repo. Briefly explain about your project what technologies you used and something else with 2-3 sentences. 
![chrome-capture (21)](https://user-images.githubusercontent.com/58504655/93437549-8c1afe80-f8e5-11ea-91b5-334802b64ef3.jpg)

- [x]  **Optional** Please try to add your screenshot in assets/images. That would be great.
![chrome-capture (22)](https://user-images.githubusercontent.com/58504655/93437648-a81ea000-f8e5-11ea-8b82-f37f04201399.jpg)
**Please make the changes and request re-review for the given project**
Stay safe😇

### Happy Coding
------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

sinansevgi committed:
Sign In button styling changed.

zdrale committed:
Update README.md

Update README file.

zdrale committed:
Update README.md

zdrale committed:
Restyled logos

sinansevgi committed:
[Style Changes]: Heading style fixed.

We used Roboto font instead of Mint's Avenir Next font because Avenir 
Next is not free.

arslanbisharat reviewed (changes requested):
### Request changes
I can see you worked hard on this project but I am sorry to inform you that I cannot Approve this project because it is not completed yet.😟
_It's okay. We are here to learn. Don't get discouraged. Work more hard._
**Please have a look at the suggestions to get more understanding. 👀**

- [x] As you said "**On the original page, they used SVG files, not text, so we could not find the original font so we used the original SVG file that we found on mint's login page.**"  but the text is inside the SVG. In SVG we can add text. Please go to the original page and have a look again. You can either use SVG files for this project too. Please have a look at this [project ](https://rawcdn.githack.com/hemant-soni-vst-au4/HTML-forms/e426ad17f96181f9d31c0405e046bab3b6e57c84/index.html)

**Please make the changes and request re-review for the given project**
Stay safe😇

### Happy Coding
-------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

sinansevgi committed:
[Html Update]: Side logo links fixed

umairarshadbutt reviewed (approved):
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://zdrale.github.io/loginform/) shows a good understanding and usage of HTML Forms. :100:
##
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @zdrale and @sinansevgi ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know on slack at any time. I will be happy to help.**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

arslanbisharat reviewed (approved):
### Congratulations APPROVED :clap:
Hello Team! I can see you worked really hard on this project. I reviewed your project  and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
Now you can merge the branches and move forward with your curriculum. 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 
**Happy Coding**:smiley:
-------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) @slack: arslanbisharat
